### PR TITLE
Add simplified migration doc

### PR DESCRIPTION
**Pull request by evanphx**

### CODE REVIEW

File: content/source/docs/enterprise/private/migrate.html.md [hunk 1]
```diff
@@ -31,6 +31,17 @@ To revert to the AMI after running the migration script:
 * If the EC2 instance is still running from the migration process, run `shutdown` on it to get a new instance created.
 * Return to the original hostname used for the cluster.
 
+## Simple Upgrade (Recommended)
+
+For users that did not significantly change the reference terraform modules used to deploy
```
Comment (maryelizbeth) [inline content/source/docs/enterprise/private/migrate.html.md]: s/terraform/Terraform

File: content/source/docs/enterprise/private/migrate.html.md [hunk 2]
```diff
@@ -31,6 +31,17 @@ To revert to the AMI after running the migration script:
...
+the AMI, there is a simplified migration procedure that can be used.
+
+For users that modified the terraform modules, you're free to follow the simplified steps 
+as well, but you'll likely need to modify some terraform modules again to match the modifications
+you did in the past.
```
Comment (maryelizbeth) [inline content/source/docs/enterprise/private/migrate.html.md]: you did --> you made

File: content/source/docs/enterprise/private/migrate.html.md [hunk 3]
```diff
@@ -31,6 +31,17 @@ To revert to the AMI after running the migration script:
...
+
+If you want to do this procedure, head to the [Simplified Migration Page](./simplified-migration.html).
```
Comment (maryelizbeth) [inline content/source/docs/enterprise/private/migrate.html.md]: head to the --> please follow the steps outlined on

File: content/source/docs/enterprise/private/simplified-migration.html.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+The migrator will prompt you to run again with the `-confirm` flag to confirm that you wish to run this step, because it
+shuts down operations on the current instance. The shutdown is required to ensure consistency of the data.
+To confirm the shutdown and proceed, run:
+
+```
+$ sudo ./migrator -confirm
+```
+
+Next, the migration will move the data into PostgreSQL and output the encryption password
```
[the remark this answers is not in the document]
Comment (evanphx) [reply content/source/docs/enterprise/private/simplified-migration.html.md]: Ah, the vault data that is stored outside of PostgreSQL in the AMI.

File: content/source/docs/enterprise/private/simplified-migration.html.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+   jIQzYXtNZNKfkyHbj2WHqTkrSQPkeX8gCrfPAkJHZ5s
+
+```
+
+
+The value of the encryption password (`jIQzYXtNZNKfkyHbj2WHqTkrSQPkeX8gCrfPAkJHZ5s` here) will be different for your run of the
+migrator tool. Copy the provided value and add it to your `.tfvars` file like this:
+
+```
+encryption_password = "jIQzYXtNZNKfkyHbj2WHqTkrSQPkeX8gCrfPAkJHZ5s<Paste>
```
[the remark this answers is not in the document]
Comment (evanphx) [reply content/source/docs/enterprise/private/simplified-migration.html.md]: A vim paste bug. Should be removed.

**Pull request merged**